src/fparser/common/tests/test_sourceinfo.py

```diff
@@ -94,23 +94,24 @@ def test_fortranformat_constructor(pretty):
     assert unit_under_test.is_fix == (not pretty[0] and not pretty[1])
     assert unit_under_test.is_pyf == (pretty[0] and pretty[1])
     assert unit_under_test.mode == pretty[2]
+    assert not unit_under_test.f2py_enabled
 
 
 ##############################################################################
 @pytest.fixture(scope="module",
-                params=[(False, False),
-                        (False, True),
-                        (True, False),
-                        (True, True)])
-def permutations(request):
+                name="permutations",
```

Comment:
This appears to only be used in one test. If there are no plans to use it anywhere else then perhaps it would be better to implement with `parametrize`?

Reply:
Good spot. I think this pre-dates our discovery of `parametrize`.